backend/server/accounts/tests.py

```diff
@@ -46,3 +48,35 @@ def test_login(self):
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(author_data["displayName"], 'testLogin')
+    
+    def test_edit(self):
```

Comment:
this tests creation not edit, so probably a good idea to rename this to avoid confusion when testing edit later.

Reply:
This does change edit though. It changes the authour GitHub and displayname. Then the signup would be creation right?

Reply:
Ah, I see. I was confused, cause there should to be a separate endpoint for PUT/DELETE that includes the id in the url i.e. /author/1. I'll resolve this since that is a separate issue, and instead will fix the API endpoint in a different PR.